server/routes.py

```diff
@@ -56,6 +59,7 @@ def analyse():
         while count_report_to_analyse() > 0:
             map_calculate(batch_size)
 
+        group_reports_by_location()
```

Comment:
Should we do batch-processing before http response? Maybe respond and start a backgroud job with processing?

Reply:
You mean like a cron job?

Reply:
Just run `analysis/scripts/05_script_analysis.py` then